.github/workflows/test.yml

```diff
@@ -1,25 +1,74 @@
 name: Build and Test
 on:
   pull_request:
-    paths:
-      # Only run on changes to files that are actually included in the build or this file itself
-      - "code/code/**.cpp"
-      - "code/code/**.cc"
-      - "code/code/**.h"
-      - ".github/workflows/test.yml"
 
 jobs:
   build_and_test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Going to fail after an incompatible update. Perhaps pin it to 22.04? Or are we fine with an occasional breakage?

Reply:
For some reason I was under the impression that ubuntu-latest and ubuntu-20.04 were the only ones available, but after looking ubuntu-22.04 does exist, so I've switched them over to it. Good call.